deploy/os/controller.yaml

```diff
@@ -0,0 +1,72 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: che-workspace-controller
+  namespace: che-workspace-controller
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: che-workspace-controller
+  template:
+    metadata:
+      labels:
+        app: che-workspace-controller
+    spec:
+      serviceAccountName: che-workspace-controller
+      containers:
+        - name: che-workspace-controller
+          image: quay.io/che-incubator/che-workspace-controller:nightly
+          imagePullPolicy: Always
+          env:
+            - name: WATCH_NAMESPACE
+              value: ""
+            - name: POD_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.name
+            - name: OPERATOR_NAME
+              value: "che-workspace-operator"
+          ports:
+            - name: webhook-server
+              containerPort: 8443
+          volumeMounts:
+            - name: webhook-tls-certs
+              mountPath: /tmp/k8s-webhook-server/serving-certs
```

Comment:
Consider duplicating it into env var and then use the value here https://github.com/che-incubator/che-workspace-operator/blob/2dc7f2e252f7912548fa405dac1d24cfda30e9fb/pkg/webhook/server/server.go#L26
it would make a bit transparent that the controller is configured with this value.
@amisevsk WDYT?

Reply:
I'm not sure I understand fully, but I feel like reading the value from a mounted file is the more canonical approach; that's what's done in e.g. openshift oauth proxy:
```
      volumeMounts:
        - mountPath: /etc/tls/private
          name: proxy-tls
...
  volumes:
    - name: proxy-tls
      secret:
        defaultMode: 420
        secretName: proxy-tls
```

Reply:
I just meant here moving hard-coded path to an environment variable, and we would be able to have:
```
     env:
       - name: webhooks.certs.path 
         value: /tmp/k8s-webhook-server/serving-certs
...
      volumeMounts:
        - mountPath: /tmp/k8s-webhook-server/serving-certs
          name: webhook-tls
...
  volumes:
    - name: webhook-tls
      secret:
        secretName: webhook-tls
```
but it's not critical